static/css/main.css

```
@@ -142,74 +358,99 @@ button:disabled {
 }
 
 .finding-card.filtered-out {
     display: none;
 }
 
+.finding-card code,
+.finding-card pre,
+#attackStatus,
+#scanUrl {
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)